mono/utils/mono-complex.h

```diff
@@ -11,24 +11,34 @@
 
 #include <config.h>
 #include <glib.h>
-
-#if !defined (HAVE_COMPLEX_H) || (defined (ANDROID_UNIFIED_HEADERS) && __ANDROID_API__ < 23)
-#include <../../support/libm/complex.h>
-#else
-#include <complex.h>
-#endif
-
 #define _USE_MATH_DEFINES // needed by MSVC to define math constants
 #include <math.h>
 
-#ifdef _MSC_VER
+#define double_complex	mono_double_complex
+#define creal 		mono_double_complex_creal
+#define cimag 		mono_double_complex_cimag
+#define cabs 		mono_double_complex_cabs
```

Comment:
We talked off-GitHub and @jaykrell will try to remove these #defines and update `threadpool-worker-default.c` to use `mono_double_complex_creal` etc.  Otherwise, LGTM.